### ISSUE TITLE
In runtime.py, the signature definition for callbackEndOfSystemSizing seems incorrect

### ISSUE THREAD

**Issue by hdou** [NREL/EnergyPlus#9481]
Issue overview
--------------
In runtime.py, the signature definition for callbackEndOfSystemSizing seems incorrect

It's:
```
        self.api.callbackEndOfZoneSizing.argtypes = [c_void_p, self.py_state_callback_type]
        self.api.callbackEndOfZoneSizing.restype = c_void_p
        self.api.callbackEndOfSystemSizing.argtypes = [self.py_state_callback_type]
        self.api.callbackEndOfSystemSizing.restype = c_void_p
```

Should it be like below?
```
        self.api.callbackEndOfZoneSizing.argtypes = [c_void_p, self.py_state_callback_type]
        self.api.callbackEndOfZoneSizing.restype = c_void_p
        self.api.callbackEndOfSystemSizing.argtypes = [c_void_p, self.py_state_callback_type]    # <== this line
        self.api.callbackEndOfSystemSizing.restype = c_void_p
```

**Comment by Myoldmopar** [comment]
Yep, seems like it, good catch!  As you've already deduced, users looking for a workaround can just apply the same change you've noted here.  I'll fix this up in a commit for the next E+ version.  Thanks!

**Comment by Myoldmopar** [comment]
Looks like the same thing happened with `api.registerExternalHVACManager`

**Comment by Myoldmopar** [comment]
Fix is in a PR, should be merged soon, and this will be addressed in 22.2.  Thank you for reporting this!
